docker-compose.yml

```diff
@@ -1,16 +1,21 @@
-version: '2'
-
+version: '2.1'
 services:
-  postgres:
-    image: postgres:9.4
-    ports:
-      - "5432"
+  postgresql:
+    image: docker.io/bitnami/postgresql:11
```

Comment:
What is the reason for not using the official postgres repository in favour of Bitnami one? (just curious)

Reply:
Honestly in this case no reason, but I used to use `bitnami/postgresql` image instead of the official one because it allow me to set a lot of Postgres configurations using docker env without the need to bind any files

Reply:
I prefer binding configuration to ENV vars. I dislike ENV var side-effect configuration.

Please keep the official postgres container image.